_maps/WIP/Z.01.Desert-Dam.dmm

```diff
@@ -3538,8 +3528,8 @@
 /area/desert_dam/interior/lab_northeast/biology)
 "alh" = (
 /obj/structure/table/reinforced,
-/obj/item/device/defibrillator,
-/obj/item/device/defibrillator,
+/obj/item/defibrillator,
+/obj/item/defibrillator,
```

Comment:
Did La's PR  not properly remove the `device` paths from the maps?

Reply:
they didn't remove it from the desert dam, but did for the rest.